assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1558,10 +1559,18 @@ def apply_annotation(
         SIMILARITY: [prompt_column_name, completion_column_name, ground_truth_column_name]
     }
     # Sampling
-    production_df = production_df.sample(withReplacement=False, fraction=sample_rate)
-    production_df_with_index = production_df.withColumn("id",
-                                                        row_number()
-                                                        .over(Window.orderBy(monotonically_increasing_id()))-1)
+    production_df_sampled = production_df.sample(withReplacement=False, fraction=sample_rate)
+    if production_df_sampled.count() == 0:
+        print("Not enough data resulting from sample_rate and production dataset. "
+              "Using default sampling value of 5. To use custom sample_rate with this dataset, "
```

Comment:
maybe mention that we just pick 5 from head (that it is not a randomly sampling)?